src/main/java/com/icthh/xm/tmf/ms/activation/events/bindings/MessagingConfiguration.java

```diff
@@ -1,197 +1,35 @@
-    }
-
-    public static String buildChanelName(String tenantKey) {
-        return SAGA_EVENTS_PREFIX + tenantKey;
-    }
-
-    private void createChannels(String tenantName) {
-        try {
-            String tenantKey = upperCase(tenantName);
-            createHandler(buildChanelName(tenantKey), "activation", tenantKey, earliest);
-        } catch (Exception e) {
-            log.error("Error create channels for tenant " + tenantName, e);
-            throw e;
-        }
-    }
-
-    private synchronized void createHandler(String chanelName, String consumerGroup, String tenantName,
-                                            StartOffset startOffset) {
-        if (!channels.containsKey(chanelName)) {
-
-            log.info("Create binding to {}. Consumer group {}", chanelName, consumerGroup);
-
-            KafkaBindingProperties props = new KafkaBindingProperties();
-            props.getConsumer().setAutoCommitOffset(false);
-            props.getConsumer().setAutoCommitOnError(false);
-            props.getConsumer().setStartOffset(startOffset);
-            kafkaExtendedBindingProperties.setBindings(Collections.singletonMap(chanelName, props));
-
-            ConsumerProperties consumerProperties = new ConsumerProperties();
-            consumerProperties.setMaxAttempts(Integer.MAX_VALUE);
-            consumerProperties.setHeaderMode(HeaderMode.none);
-            consumerProperties.setPartitioned(true);
-            consumerProperties.setConcurrency(applicationProperties.getKafkaConcurrencyCount());
-
-            ProducerProperties producerProperties = new ProducerProperties();
-            producerProperties.setPartitionKeyExtractorName(TASKS_PARTITION_KEY_EXTRACTOR_STRATEGY);
-            producerProperties.setPartitionCount(kafkaBinderConfigurationProperties.getMinPartitionCount());
-
-            BindingProperties bindingProperties = new BindingProperties();
-            bindingProperties.setConsumer(consumerProperties);
-            bindingProperties.setProducer(producerProperties);
-            bindingProperties.setDestination(chanelName);
-            bindingProperties.setGroup(consumerGroup);
-            bindingServiceProperties.setBindings(Collections.singletonMap(chanelName, bindingProperties));
-
-            SubscribableChannel channel = bindingTargetFactory.createInput(chanelName);
-            bindingService.bindConsumer(channel, chanelName);
-
-            HealthIndicatorRegistry registry = bindersHealthIndicator.getRegistry();
-            if (registry.get(KAFKA) == null) {
-                bindersHealthIndicator.getRegistry().register(KAFKA, kafkaBinderHealthIndicator);
-            }
-
-            channels.put(chanelName, channel);
-
-            channel.subscribe(message -> {
-                try {
-                    MdcUtils.putRid(MdcUtils.generateRid() + ":" + tenantName);
-                    sleuthWrapper.runWithSleuth(message, () -> handleEvent(tenantName, message));
```

Comment:
pls check that this still work in new implementation

Reply:
tested
![photo_2025-01-27_11-01-28](https://github.com/user-attachments/assets/578abe64-585f-41c5-80cf-35a29456b2cf)
![photo_2025-01-27_11-01-22](https://github.com/user-attachments/assets/a1130b31-a5ca-4110-8e58-69842bb62e6c)